packages/aws-cdk-lib/aws-lambda-event-sources/lib/kinesis.ts

```diff
@@ -12,15 +14,19 @@ export interface KinesisEventSourceProps extends StreamEventSourceProps {
   readonly startingPositionTimestamp?: number;
 }
 
-/**
- * Use an Amazon Kinesis stream as an event source for AWS Lambda.
- */
-export class KinesisEventSource extends StreamEventSource {
+interface KinesisSource {
```

Comment:
Can you provide a docstring for this new interface?

Reply:
I can add something, I just didn't initially add, as both `KinesisSource` and `KinesisEventSourceBase` are not public

Reply:
oh I see, it's not exported. That should be fine then.